bezalel001/Maintenance-Tracker
# feat(detail): implements request detail pages for admin and user

**What does this PR do?**
Allows user to view details of a maintenance request
Aloows admin to view details of a maintenance request and approve, reject or resolve a request

**Description of Task to be completed?**
A maintenance detail page that allows user to view the detail description of a request and the current status of the request, i.e whether the request has been seen, approved, rejected of resolved.

**How should this be manually tested?**
Visit https://bezalel001.github.io/Maintenance-Tracker/index.html and click on the links:
i) Request detail
ii) Request detail-admin
to see the UIs

**Any background context you want to provide?**
nil

**What are the relevant pivotal tracker stories?**
User should be able to view the details of a request 
ID: #157419459
